flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/FlinkDynamicTableFactory.java

```diff
@@ -118,7 +118,7 @@ public DynamicTableSink createDynamicTableSink(Context context) {
               catalogTable, tableProps, objectPath.getDatabaseName(), objectPath.getObjectName());
     }
 
-    return new IcebergTableSink(tableLoader, tableSchema, context.getConfiguration());
+    return new IcebergTableSink(tableLoader, tableSchema, context.getConfiguration(), tableProps);
```

Comment:
similarly, can we rename `tableProps` here to `writeProps`?

Reply:
done